incubator/group/types.proto

```diff
@@ -109,11 +109,16 @@ message StdDecisionPolicy {
         ThresholdDecisionPolicy threshold = 1;
     }
 }
-
+// todo: we need to ensure that ThresholdDecisionPolicy can not be modified during a running proposal
 message ThresholdDecisionPolicy {
-    string threshold = 1 [(gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec"];
-    google.protobuf.Duration min_voting_window = 2;
-    google.protobuf.Duration max_voting_window = 3;
+    // threshold is a fix weight value that must be exceeded for a proposal to succeed.
+    string threshold = 1 [(gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec", (gogoproto.nullable) = false];
+
+    // TODO: why do we need min/max? when people can change their vote there is no way we can
+    // close a proposal early. Even if everybody has voted already. I would propose a single value instead so that
+    // we can calculate the end of the voting period.
```

Comment:
Why can't a proposal be closed early? We can just have it that at any point after `min_voting_window`, `MsgExec` can be submitted and a proposal could be closed. Is that a problem?

Reply:
I should have been more precise: implementation wise **it is not a problem** to allow `MsgExec` after `min_voting_window` on the server side. The challenge for me is the communication/ UX problem when votes or changing votes are allowed between min and end of voting period. The MsgExec could be executed any time. Even in the same block can be votes before and after the MsgExec that would have different effect. As a voter I would not understand why my vote was rejected when election was open a sec before. As a client developer I would need to query the proposal status in this period more often to enable/disable voting.
I prefer a clear end timestamp instead that allows votes only in blocks before that timestamp but I am not aware of all the use cases.